doc/introduction/interfaces/numpy.rst

```diff
@@ -108,8 +108,8 @@ with respect to both QNode parameters ``phi`` and ``theta``:
 
 .. code-block:: python
 
-    phi = np.array([0.5, 0.1])
-    theta = 0.2
+    phi = np.array([0.5, 0.1], requires_grad=True)
+    theta = np.array(0.2, requires_grad=True)
```

Comment:
Wouldn't the output on line 118 change too?

Reply:
I get

(array([-0.47942554,  0.        ]), array(-6.9388939e-18))
which is essentially the same thing.